PWGCF/TwoParticleCorrelations/Tasks/CMakeLists.txt

```diff
@@ -63,6 +63,11 @@ o2physics_add_dpl_workflow(di-hadron-cor
                            PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2Physics::PWGCFCore
                            COMPONENT_NAME Analysis)
 
+o2physics_add_dpl_workflow(gen-corr
```

Comment:
Could the task name (and source file) be a bit more specific?
`gen` immediately addresses MC generator level which I guess it is not the aim
And `corr` is ambiguous having in mind the task is whithin the `TwoParticleCorrelations` directory
As it seems that the FT0 is being considered, what about something like `long range correlations`?